itunesSearch/itunesSearch/ResultType.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  ResultType.swift
+//  itunesSearch
+//
+//  Created by Bradley Yin on 8/6/19.
+//  Copyright © 2019 bradleyyin. All rights reserved.
+//
+
+import Foundation
+enum ResultType: String{
```

Comment:
I would suggest placing a space between the `import` and the start of real code.

Reply:
(a new line space I mean)